lib/token.h

```diff
@@ -959,19 +959,49 @@ class CPPCHECKLIB Token {
      */
     static void eraseTokens(Token *begin, const Token *end);
 
+    /**
+     * Insert new token after this token. This function will handle
+     * relations between next and previous token also.
+     * @param tokenStr String for the new token.
+     * the first one on the tokens list.
+     */
+    RET_NONNULL Token* insertToken(const std::string& tokenStr)
+    {
+        return insertToken(tokenStr, false);
+    }
     /**
      * Insert new token after this token. This function will handle
      * relations between next and previous token also.
      * @param tokenStr String for the new token.
      * @param originalNameStr String used for Token::originalName().
      * the first one on the tokens list.
```

Comment:
Leftover from the previous PR

Reply:
I left documentation on each function because they have different parameter lists. I could adjust it, so only the one with all the parameters is documented.

Reply:
I was mostly thinking about the orphaned line.

Reply:
Oh, I missed that. Done.